test/parseDetectorTables.test.ts

```diff
@@ -0,0 +1,26 @@
+const vsCodeIntegration: string = 'Latest time seen by detector. To be used in VSCode integration.';
+suite('Detector Dataset Parser', () => {
+    test('Find table by column name', async () => {
+        const table: detectorTable | undefined = findTableByColumnName(detectorDataset.properties.dataset, ColumnName.failureCount);
+        assert.equal(table, detectorDataset.properties.dataset[4].table);
+    });
+
+    test('Find table by row value', async () => {
+        const table: detectorTable | undefined = findTableByRowValue(detectorDataset.properties.dataset, vsCodeIntegration);
+        assert.equal(table, detectorDataset.properties.dataset[1].table);
+    });
+
+    test('Get values by column name', async () => {
+        const timestamp: string = "7/26/2019 8:40:00 PM UTC ";
+        const table: detectorTable | undefined = findTableByRowValue(detectorDataset.properties.dataset, vsCodeIntegration);
+        assert.ok(table);
+        const values: string[] = getValuesByColumnName(nonNullValue(table), ColumnName.value);
```

Comment:
How about a unit test for `getValuesByColumnName` for a column that returns multiple values